src/rgen/Microsoft.Macios.Bindings.Analyzer/Resources.resx

```diff
@@ -1,190 +1,260 @@
 <?xml version="1.0" encoding="utf-8"?>
-
 <root>
-    <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
```

Comment:
Looks like this entire file got reformatted?

Reply:
I think it was the auto-formatter that did it.